bottles/frontend/views/bottle_dependencies.py

```diff
@@ -63,8 +63,8 @@ def __init__(self, details, config: BottleConfig, **kwargs):
         self.btn_report.connect("clicked", open_doc_url, "contribute/missing-dependencies")
         self.btn_help.connect("clicked", open_doc_url, "bottles/dependencies")
 
-        if self.manager.utils_conn.status == False:
-            self.stack.set_visible_child_name("page_offline")
+        if not self.manager.utils_conn.status:
+            self.stack.set_visible_child_name("page_offline") 
 
```

Comment:
```suggestion
            self.stack.set_visible_child_name("page_offline")
```
Remove space at end of line
```suggestion
            self.stack.set_visible_child_name("page_offline")
```
```suggestion
            self.stack.set_visible_child_name("page_offline")
```